workers/unity/Packages/io.improbable.worker.sdk.mobile/BuildPostProcessXCode.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using UnityEditor;
+using UnityEditor.Callbacks;
+
+namespace Improbable.Gdk.Mobile
+{
+    public static class BuildPostProcessXCode
+    {
+        private static Type pbxType;
+
+        [PostProcessBuild]
+        public static void OnPostProcessBuild(BuildTarget buildTarget, string path)
+        {
+            if (buildTarget != BuildTarget.iOS)
+            {
+                return;
+            }
+
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
+            foreach (var assembly in assemblies)
+            {
+                var type = assembly.GetTypes().FirstOrDefault(t => t.FullName == "UnityEditor.iOS.Xcode.PBXProjectddd");
```

Comment:
`PbxProjectddd`?

Reply:
On purpose to make buildkite fail.. but its not failing :/